libraries/Microsoft.Bot.Builder.AI.QnA/Utils/QnACardBuilder.cs

```diff
@@ -77,48 +77,82 @@ public static IMessageActivity GetSuggestionsCard(List<string> suggestionsList,
         }
 
         /// <summary>
-        /// Get active learning suggestions card.
+        /// Get Card for MultiTurn scenario. (Can be deprected from 4.10.0 release of sdk).
         /// </summary>
         /// <param name="result">Result to be dispalyed as prompts.</param>
-        /// <param name="cardNoMatchText">No match text.</param>
+        /// <param name="cardNoMatchText">No match text.</param>       
```

Comment:
I don't know what this means. Consider improving the quality of the doc comment.

Reply:
Updated content.

---
In reply to: [430557111](https://github.com/microsoft/botbuilder-dotnet/pull/3935#discussion_r430557111) [](ancestors = 430557111)